roles/cinder-common/tasks/ceph_integration.yml

```diff
@@ -1,21 +1,40 @@
 ---
-- name: fetch rados.py
-  get_url:
-    url: "{{ ceph.rados_url }}"
-    dest: /opt/openstack/current/cinder/lib/python2.7/site-packages/rados.py
+- name: import rados and rbd modules into cinder
+  file:
+    src: /usr/lib/python2.7/dist-packages/{{ item }}
+    dest: /opt/openstack/current/cinder/lib/python2.7/site-packages/{{ item }}
+    state: link
     owner: root
     group: root
     mode: 0644
-    timeout: 30
+  with_items:
+    - rados.so
+    - rbd.so
+  when: ceph.stable_release != 'hammer'
 
-- name: fetch rbd.py
-  get_url:
-    url: "{{ ceph.rbd_url }}"
-    dest: /opt/openstack/current/cinder/lib/python2.7/site-packages/rbd.py
+- name: remove modules those were used for Hammer
+  file:
+    path: /opt/openstack/current/cinder/lib/python2.7/site-packages/{{ item }}
+    state: absent
+  with_items:
+    - rados.py
+    - rados.pyc
+    - rbd.py
+    - rbd.pyc
+  when: ceph.stable_release != 'hammer'
+
+- name: import rados and rbd modules into cinder
```

Comment:
We can remove this task if we don't need to support installation of hammer version in the future.
Should I remove this?